src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -72,7 +73,8 @@
 
 .monaco-workbench .editor-instance .xterm-decoration-overview-ruler,
 .monaco-workbench .pane-body.integrated-terminal .xterm-decoration-overview-ruler {
-	z-index: 31; /* Must be higher than .xterm-viewport */
+	/* Must be higher than .xterm-viewport */
+	z-index: 31;
```

Comment:
I prefer the old way as it indicates the comment is unambiguously about the one line, when the comment is above it's unclear whether the comment is referring to all lines below or just the single line